README.md

```diff
@@ -80,7 +80,74 @@ These are the cli options exposed to `pytest` by the plugin.
 | `--snapshot-warn-unused`       | Prints a warning on unused snapshots rather than fail the test suite.               | `False`                                          |
 | `--snapshot-default-extension` | Use to change the default snapshot extension class.                                 | `syrupy.extensions.amber.AmberSnapshotExtension` |
 
-### Built-In Extensions
+### Assertion Options
+
+These are the options available on the `snapshot` assertion fixture.
+Use of these options are one shot and do not persist across assertions.
+For more persitent options see the [advanced usage](#advanced-usage).
+
+#### `matcher`
+
+This allows you to match on a property path and value to control how specific object shapes are serialized.
+
+The matcher is a function that takes two keyword arguments:
+
+| Argument | Description                                                                   |
+| -------- | ----------------------------------------------------------------------------- |
+| `data`   | Current serializable value being matched on                                   |
+| `path`   | Ordered path traversed to the current value e.g. `(("a", dict), ("b", dict))` |
+
+**NOTE:** Do not mutate the value received as it could cause unintended side effects.
+
+##### Built-In Matchers
+
+Syrupy comes with a few built-in matcher presets that can be used to make easy work of using property matchers.
```

Comment:
You mention a "few". Are they all documented here? Path_type is more of a matcher factory

Reply:
There's only one so far, will update the wording